packages/client/bin/cli.ts

```diff
@@ -149,7 +148,7 @@ async function run() {
     common,
     syncmode: args.syncmode,
     lightserv: args.lightserv,
-    datadir: `${os.homedir()}/Library/Ethereum/ethereumjs`,
+    datadir: args.datadir,
```

Comment:
Ah, can we just leave this on the old value (maybe my answer to your question was a bit misleading here)? Think this was a reasonable default and ensures that the data directory from "serious" client runs is different from the one used for the tests. Otherwise the test runs would override an already downloaded longer chain from the normal CLI runs.

Or do you have some other specific reason for this change?

If you are not satisfied that this is so os-specific another option would be to tackle https://github.com/ethereumjs/ethereumjs-vm/issues/999 opened by Paul and add more appropriate os-specific default directories. Maybe we should add this anyhow to the list of TODOs before a first more public release.

Yet another very simple option to avoid conflicts with the test datadir: we could add a separate default with a relative directory path (to already make it somewhat more os-friendly) directly here.

Reply:
ah ok hm, well the reason I targeted this change was because the cli `--datadir` arg wasn't being used, so all client runs would use this default of `${os.homedir()}/Library/Ethereum/ethereumjs`, and I couldn't run two clients at once.

I see your point about a safe datadir for test suite runs though. I will fix.
My current thinking is to change:

```
    datadir: {
      describe: 'Data directory for the blockchain',
      default: Config.DATADIR_DEFAULT,
    },
```

to `default: ${os.homedir()}/Library/Ethereum/ethereumjs` and that should fix this problem, so a `--datadir` arg can override.

Then `Config.DATADIR_DEFAULT` can stay as `./datadir` for safe test executions.

How does that sound?

Reply:
Yeah, sounds good! 😀

Reply:
I have added this change, will come along next push that also rebases on latest master.

So `cli.ts` default datadir arg will be `${os.homedir()}/Library/Ethereum/ethereumjs`, which can be modified on start with `--datadir` flag.

And test runs should continue to use `./datadir` (`Config.DATADIR_DEFAULT` in `config.ts` when `options.datadir` is undefined).